src/main/java/core/basesyntax/Ball.java

```diff
@@ -0,0 +1,30 @@
+package core.basesyntax;
+
+public class Ball {
+    private String color;
+    private int number;
+
```

Comment:
it is a good practice to create already valid object
so it would be better to add constructor with all fields to our class)